src/rgw/rgw_op.h

```diff
@@ -1768,14 +1768,10 @@ class RGWSetAttrs : public RGWOp {
 
 class RGWGetObjLayout : public RGWOp {
 protected:
-  RGWRados::Object *target{nullptr};
   RGWObjManifest *manifest{nullptr};
   rgw_raw_obj head_obj;
 
 public:
-  RGWGetObjLayout() {
```

Comment:
Please don't remove something that is not related to this particular fix. Please don't remove the constructor.